### PR TITLE
Fixes for keeping swagger working

### PR DESCRIPTION
**Pull request by pwaszkis**

### CODE REVIEW

**Comment by tmotyl** [comment]
Hi @pwaszkis 
Please describe the issue/feature this PR is solving.

**Comment by pwaszkis** [comment]
@tmotyl After installing the module I had on the swagger page the following error:
image
This PR solving this issue.

File: Api/CategoryRepositoryInterface.php [hunk 1]
```diff
@@ -10,29 +10,29 @@ interface CategoryRepositoryInterface
 {
 
     /**
-     * @return CategoryInterface[]
+     * @return \Macopedia\Allegro\Api\Data\CategoryInterface[]
```
Comment (fsw) [inline Api/CategoryRepositoryInterface.php:13]: Why adding this FQN?

File: Api/ParameterDefinitionRepositoryInterface.php [hunk 2]
```diff
@@ -2,23 +2,21 @@
 
 namespace Macopedia\Allegro\Api;
 
-use Macopedia\Allegro\Api\Data\ParameterDefinitionInterface;
-use Macopedia\Allegro\Api\Data\ParameterInterface;
```
Comment (fsw) [inline Api/ParameterDefinitionRepositoryInterface.php:6]: Why using FQN here?

**Comment by cieslix** [comment]
fixed in https://github.com/macopedia/magento2-allegro/commit/25c464d5c58b507c4a400c1776a65427ee53fbd6

**Pull request closed without merging**